### ISSUE TITLE
double-normal function in selShapes using 0-1 range for parameters 5 and 6

### ISSUE THREAD

**Issue by iantaylor-NOAA** [r4ss/r4ss#289]
The most recent change to /inst/Shiny/Selectivities/server.R in commit 314d7aa3832708c67b8112c8a7e488e9ee6429c0 by states 
`#Input for double normal parameters: all parameters on the scale the user enters in SS` but parameters 5 and 6 are still using 0-1 range.

It seems that lines 16-30 https://github.com/r4ss/r4ss/blob/4b9457ba2a89154447a0693411a6407a714f4237/inst/Shiny/Selectivities/server.R#L16-L30
are inconsistent with lines 38-59 of that function https://github.com/r4ss/r4ss/blob/4b9457ba2a89154447a0693411a6407a714f4237/inst/Shiny/Selectivities/server.R#L38-L59

The 0-1 range is more convenient for a widget, but less helpful for parameterizing SS.
@Andrea-Havron and @allanhicks, let me know if I'm missing something.

**Comment by Andrea-Havron** [comment]
Yes, @iantaylor-NOAA, lines 16-30 are currently inconsistent with lines 38-59. @allanhicks and I decided to leave the SS relevant conditionals in place for future updates. The challenge is that the widgets for init and final on the logit scale are difficult to use. I suggest either:

- Keep the widgets for init and final on the 0-1 scale but change the numeric inputs to the logit scale
or
- Remove the widgets for init and final altogether and only accept numeric inputs for these two parameters on the logit scale

**Comment by iantaylor-NOAA** [comment]
Thanks @Andrea-Havron that makes sense. How about a third, even more complex option?

I have no idea if this can be done easily in Shiny, but the workflow is SS would really be best mimicked by a check box selecting whether to use each of the parameters 5 and 6. If the box is empty, the widget would go away or not have any impact and the numeric input would be set to -999. If the box was checked, the widget would work and have bounds something like -10 to 10.
